library/src/main/java/com/devbrackets/android/exomedia/util/DeviceUtil.kt

```diff
@@ -20,44 +20,18 @@ import android.app.UiModeManager
 import android.content.Context
 import android.content.res.Configuration
 import android.os.Build
-import java.util.*
 
 /**
  * A Utility class to help determine characteristics about the device
  */
 class DeviceUtil {
-    companion object {
-        protected val NON_COMPATIBLE_DEVICES: MutableList<Device>
 
-        init {
-            NON_COMPATIBLE_DEVICES = LinkedList()
-            NON_COMPATIBLE_DEVICES.add(Device("Amazon"))
-        }
-    }
+    private val NON_COMPATIBLE_DEVICES = listOf(Device("Amazon"))
 
     fun supportsExoPlayer(context: Context): Boolean {
```

Comment:
Long run we should make the `DeviceUtil` and interface and this the default implementation to help with customization (was a non-final class in Java to do this). I don't expect that to be done in this PR but I do want to make sure we do it before 5.0 is released

Reply:
How would the interface look like? Could it be something like this?
``interface DeviceUtil {
    fun supportsExoPlayer(context: Context): Boolean
}
``

Reply:
Pretty much though I would probably remove the context as an arg and let that be an implementation detail